meilisearch-http/src/routes/tasks.rs

```diff
@@ -288,8 +351,42 @@ async fn get_task(
     filters.uid = Some(vec![task_id]);
 
     if let Some(task) = index_scheduler.get_tasks(filters)?.first() {
-        Ok(HttpResponse::Ok().json(task))
+        let task_view = TaskView::from_task(&task);
+        Ok(HttpResponse::Ok().json(task_view))
     } else {
         Err(index_scheduler::Error::TaskNotFound(task_id).into())
     }
 }
+
+fn filter_out_inaccessible_indexes_from_query<const ACTION: u8>(
```

Comment:
In the future I think we could call this function in a lot of places

Reply:
Wow, then we should probably think about a shorter name!